src/PhotometryFit.cc

```diff
@@ -251,32 +259,40 @@ void PhotometryFit::saveChi2MeasContributions(std::string const &baseName) const
             double residual = flux - fittedStar->getFlux();
             double chi2Val = std::pow(residual / sigma, 2);
 
+            ofile << std::setprecision(9);
```

Comment:
As I understand it, if you don't give more specific instructions, then stream output uses either floating-point or scientific notation (equivalent to `printf("%.9g")`). Are 9 digits desirable for both types of output?

Reply:
Considering this output is mostly (entirely?) used for debugging, I think so?